### ISSUE TITLE
feature request: it would be wonderful if it can specify one lonely test in a file, just like vscode-jest-runner

### ISSUE THREAD

**Issue by heavenkiller2018** [Raathigesh/majestic#209]
## Is this a bug report or a feature request?
a feature request

Please specify whether this is a feature request or a bug.

feature request: it would be wonderful if it can specify one lonely test in a file, just like vscode-jest-runner. besides , in monorepo structure, it is nessesary to specify different jest config in each project

## Version Info

- Version of Majestic:
- Version of Jest:
- Version of Node:
- Operating System:

## Reproduction Repo

If this is a bug report, please provide a minimal github repository where this mentioned issue is reproducible. Majestic makes certain
assumptions regarding the test setup and it's very hard to guess the issue without looking at the exact configurations that you are using.

**Comment by sktguha** [comment]
Yes , I would like to add to it, if we could see in the left index, the individual tests , instead of just the jest file, that would be awesome

**Comment by fega** [comment]
Also in webstorm there is an instant feedback of tests when they past, not until the entire test suite fails,  I'd like to help with this project.
